packages/relay-compiler-language-graphitation/src/compilerTransforms/retainConnectionDirectiveTransform.ts

```diff
@@ -0,0 +1,65 @@
+import { IRTransform } from "relay-compiler/lib/core/CompilerContext";
+import { Directive, LinkedField } from "relay-compiler/lib/core/IR";
+import { visit } from "relay-compiler/lib/core/IRVisitor";
+
+type PathWithConnectionDirective = [any[], Directive];
+
+export function retainConnectionDirectiveTransform(
```

Comment:
Add comment about need for this retaining of the directive, which is because relay-compiler by default will strip out all of its client-specific directives. Which normally makes sense, as these should not be sent to the schema, but in our case we want to keep this directive as we will send it to Apollo Client instead.